### PR TITLE
PXC-4489 8.4.0-1 release notes and release note tasks

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   docs/release-notes/8.4.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

Release notes:
[https://pxc-docs.onrender.com/release-notes/8.4.0-1.html](https://pxc-docs.onrender.com/release-notes/8.4.0-1.html)

Render build: [https://dashboard.render.com/static/srv-crff575ds78s73cluf2g/deploys/dep-crff57dds78s73cluf80](https://dashboard.render.com/static/srv-crff575ds78s73cluf2g/deploys/dep-crff57dds78s73cluf80)

### CODE REVIEW

File: docs/release-notes/8.4.0-1.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+Percona XtraDB Cluster is based on Percona Server for MySQL. Find a complete list of improvements and bug fixes in the [Percona Server for MySQL 8.4.0-1 (2024-08-28) release notes](https://docs.percona.com/percona-server/8.4/release-notes/8.4.0-1.html).
+
+## Improvements
+
+* [PXC-4428](https://perconadev.atlassian.net/browse/PXC-4428): In asynchronous replicas, using `sql_log_bin=0` disrupted the Global Transcation Identifier (GTID) consistency, causing issues with replication and tracking transactions across the system.
+
+## Bugs Fixed
+
+* [PXC-4034](https://perconadev.atlassian.net/browse/PXC-4034): In asynchronous replicas, using `sql_log_bin=0` disrupted the Global Transcation Identifier (GTID) consistency, causing issues with replication and tracking transactions across the system.
```
Comment (alina-derkach-oaza) [inline docs/release-notes/8.4.0-1.md]: Transcation -> TranSACtion

File: docs/release-notes/8.4.0-1.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
+
+## Release Highlights
+
+Percona XtraDB Cluster is based on Percona Server for MySQL. Find a complete list of improvements and bug fixes in the [Percona Server for MySQL 8.4.0-1 (2024-08-28) release notes](https://docs.percona.com/percona-server/8.4/release-notes/8.4.0-1.html).
+
+## Improvements
+
+* [PXC-4428](https://perconadev.atlassian.net/browse/PXC-4428): In asynchronous replicas, using `sql_log_bin=0` disrupted the Global Transcation Identifier (GTID) consistency, causing issues with replication and tracking transactions across the system.
```
Comment (alina-derkach-oaza) [inline docs/release-notes/8.4.0-1.md]: Transcation -> TranSACtion

File: docs/release-notes/8.4.0-1.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+* [PXC-4453](https://perconadev.atlassian.net/browse/PXC-4453): Constant and frequent changes of the flow control's parameter `gcs.fc_limit` may hang the cluster
+
+* [PXC-4179](https://perconadev.atlassian.net/browse/PXC-4179): The `performance_schema.processlist` did not show information about wsrep applier threads and rollbacker threads.
+
+* [PXC-4277](https://perconadev.atlassian.net/browse/PXC-4277): A 3-node Percona XtraDB Cluster (PXC) became inconsistent when an ALTER statement with `ALGORITHM=INPLACE` was used.
+
+* [PXC-4367](https://perconadev.atlassian.net/browse/PXC-4367): The InnoDB semaphore wait timeout caused a server exit under a heavy load. 
+
+* [PXC-4362](https://perconadev.atlassian.net/browse/PXC-4362): The node could leave the cluster when binary logging was enabled and the function was created without super privilege.
```
Comment (alina-derkach-oaza) [inline docs/release-notes/8.4.0-1.md]: Duplicated Jira. Line 39

File: docs/release-notes/8.4.0-1.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+* [PXC-4277](https://perconadev.atlassian.net/browse/PXC-4277): A 3-node Percona XtraDB Cluster (PXC) became inconsistent when an ALTER statement with `ALGORITHM=INPLACE` was used.
+
+* [PXC-4367](https://perconadev.atlassian.net/browse/PXC-4367): The InnoDB semaphore wait timeout caused a server exit under a heavy load. 
+
+* [PXC-4362](https://perconadev.atlassian.net/browse/PXC-4362): The node could leave the cluster when binary logging was enabled and the function was created without super privilege. 
+
+* [PXC-4348](https://perconadev.atlassian.net/browse/PXC-4348): The joiner node exited with Metadata Lock `BF-BF` conflict during `IST`.
+
+* [PXC-4298](https://perconadev.atlassian.net/browse/PXC-4298): The node was disconnected when using `ALTER TABLE`, including `ADD UNIQUE` in the table containing duplicate entries (Thanks to Vit Novak for reporting this issue.)
```
Comment (alina-derkach-oaza) [inline docs/release-notes/8.4.0-1.md]: Duplicated Jira. Line 21

File: docs/release-notes/8.4.0-1.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+* [PXC-4179](https://perconadev.atlassian.net/browse/PXC-4179): The `performance_schema.processlist` did not show information about wsrep applier threads and rollbacker threads.
+
+* [PXC-4277](https://perconadev.atlassian.net/browse/PXC-4277): A 3-node Percona XtraDB Cluster (PXC) became inconsistent when an ALTER statement with `ALGORITHM=INPLACE` was used.
+
+* [PXC-4367](https://perconadev.atlassian.net/browse/PXC-4367): The InnoDB semaphore wait timeout caused a server exit under a heavy load. 
+
+* [PXC-4362](https://perconadev.atlassian.net/browse/PXC-4362): The node could leave the cluster when binary logging was enabled and the function was created without super privilege. 
+
+* [PXC-4348](https://perconadev.atlassian.net/browse/PXC-4348): The joiner node exited with Metadata Lock `BF-BF` conflict during `IST`.
```
Comment (alina-derkach-oaza) [inline docs/release-notes/8.4.0-1.md]: Duplicated Jira. Line 37

File: docs/release-notes/8.4.0-1.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+* [PXC-4428](https://perconadev.atlassian.net/browse/PXC-4428): When the State Snapshot Transfer (SST) completed, the Galera Arbitrator Daemon (garbd) also terminated any SST subprocesses.
+
+* [PXC-4453](https://perconadev.atlassian.net/browse/PXC-4453): Constant and frequent changes of the flow control's parameter `gcs.fc_limit` may hang the cluster
+
+* [PXC-4179](https://perconadev.atlassian.net/browse/PXC-4179): The `performance_schema.processlist` did not show information about wsrep applier threads and rollbacker threads.
+
+* [PXC-4277](https://perconadev.atlassian.net/browse/PXC-4277): A 3-node Percona XtraDB Cluster (PXC) became inconsistent when an ALTER statement with `ALGORITHM=INPLACE` was used.
+
+* [PXC-4367](https://perconadev.atlassian.net/browse/PXC-4367): The InnoDB semaphore wait timeout caused a server exit under a heavy load.
```
Comment (alina-derkach-oaza) [inline docs/release-notes/8.4.0-1.md]: Duplicated Jira. Line 45

File: docs/release-notes/8.4.0-1.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.4.0-1.md]: Global TranSCAtion Identifier (GTID) -> Global TranSACtion Identifier (GTID)

File: docs/release-notes/8.4.0-1.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.4.0-1.md]: Global TranSCAtion Identifier (GTID) -> Global TranSACtion Identifier (GTID)

**Review by alina-derkach-oaza — APPROVED** [review]

File: docs/release-notes/8.4.0-1.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+## Improvements
+
+* [PXC-4428](https://perconadev.atlassian.net/browse/PXC-4428): In asynchronous replicas, using `sql_log_bin=0` disrupted the Global Transaction Identifier (GTID) consistency, causing issues with replication and tracking transactions across the system.
+
+## Bugs Fixed
+
+* [PXC-4034](https://perconadev.atlassian.net/browse/PXC-4034): In asynchronous replicas, using `sql_log_bin=0` disrupted the Global Transaction Identifier (GTID) consistency, causing issues with replication and tracking transactions across the system.
+
+* [PXC-4041](https://perconadev.atlassian.net/browse/PXC-4041): When running `FLUSH TABLES` after a statement was prepared, the node could exit due to broken consistency.
```
Comment (kamil-holubicki) [inline docs/release-notes/8.4.0-1.md]: Something wrong here

File: docs/release-notes/8.4.0-1.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+* [PXC-4179](https://perconadev.atlassian.net/browse/PXC-4179): The `performance_schema.processlist` did not show information about wsrep applier threads and rollbacker threads.
+
+* [PXC-4277](https://perconadev.atlassian.net/browse/PXC-4277): A 3-node Percona XtraDB Cluster (PXC) became inconsistent when an ALTER statement with `ALGORITHM=INPLACE` was used.
+
+* [PXC-4298](https://perconadev.atlassian.net/browse/PXC-4298): The node was disconnected when using `ALTER TABLE`, including `ADD UNIQUE` in the table containing duplicate entries (Thanks to Vit Novak for reporting this issue.)
+
+* [PXC-4313](https://perconadev.atlassian.net/browse/PXC-4313): When statements were executed in Replica Set Update (RSU) mode, they generated local Global Transaction Identifier (GTID) events.
+
+* [PXC-4315](https://perconadev.atlassian.net/browse/PXC-4315): There was no Buffer Flush (BF) abort, but a message saying “Metadata Lock (MDL) conflict… solved by abort” was printed.
```
Comment (kamil-holubicki) [inline docs/release-notes/8.4.0-1.md]: Brute-Force, not Bruffer Flush

File: docs/release-notes/8.4.0-1.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
+
+## Release Highlights
+
+Percona XtraDB Cluster is based on Percona Server for MySQL. Find a complete list of improvements and bug fixes in the [Percona Server for MySQL 8.4.0-1 (2024-08-28) release notes](https://docs.percona.com/percona-server/8.4/release-notes/8.4.0-1.html).
+
+## Improvements
+
+* [PXC-4428](https://perconadev.atlassian.net/browse/PXC-4428): In asynchronous replicas, using `sql_log_bin=0` disrupted the Global Transaction Identifier (GTID) consistency, causing issues with replication and tracking transactions across the system.
```
Comment (kamil-holubicki) [inline docs/release-notes/8.4.0-1.md]: Something is wrong here. This one is about garbd. Listed below in 'bugs' section as well.

**Review by kamil-holubicki — CHANGES_REQUESTED** [review]

**Review by kamil-holubicki — APPROVED** [review]

**Pull request merged**